psij-ci-setup

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+FORCE=0
+
+if [ "$1" == "-f" ]; then
+    FORCE=1
+    shift
+fi
+
+MYPATH="$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+
+echo
+echo "================================================================"
+echo "This script will install requirements for the PSI/J CI tests and"
+echo "add a cron job to run the tests once a day at some random time. "
+echo "================================================================"
+echo
+
+RESPONSE=""
+
+while [ "$RESPONSE" != "C" ] && [ "$RESPONSE" != "X" ]; do
+
+    echo -n "Would you like to (C)ontinue or E(x)it? "
+    read -n1 RESPONSE
+    echo
+    RESPONSE=${RESPONSE^}
+    
+    if [ "$RESPONSE" == "X" ]; then
```

Comment:
logic should be reversed - like, break when the user enters `?`.  BTW: this prompts for `E(x)it` (lowercase `x`) but checks for uppercase `X`.

Reply:
> logic should be reversed - like, break when the user enters `?`.

Good point.

> BTW: this prompts for `E(x)it` (lowercase `x`) but checks for uppercase `X`.

The input is converted to capital letters in line 26.

Reply:
> > logic should be reversed - like, break when the user enters `?`.
> 
> Good point.
> 
> >

Actually wait. The behavior, as it is, because of the while loop, is to keep asking if neither c or x are entered. So I think this works as intended.